### DISCUSSION TITLE
Not working openpose

### DISCUSSION THREAD

**Discussion by Wundrwaffle** [Mikubill/sd-webui-controlnet#2015] [Q&A] [answered]
As i understood its not working with 1.6 version of webui?
Without CN:
![Screenshot_5](https://github.com/Mikubill/sd-webui-controlnet/assets/117706507/f518b99b-a42c-4c14-aba9-38c897e72fd3)

And using pose:
![Screenshot_6](https://github.com/Mikubill/sd-webui-controlnet/assets/117706507/20c220ba-ed8c-49ad-a88b-18a188231b36)

Tried different preprocessors, no sense. Any idea?

**Comment by huchenlei** [comment, marked as answer]
If your input is already an openpose image, you should select none in preprocessor.
